client/my-sites/stats/info-panel/index.jsx

```diff
@@ -263,7 +263,7 @@ class StatsInfoPanel extends React.PureComponent {
 									href="http://wordpress.com/support/videos/"
 								>
 									<Gridicon icon="folder" />
-									{ this.props.translate( 'About Videos on WordPress.com' ) }
+									{ this.props.translate( 'About videos on WordPress.com' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.